R/db_oracle.R

```diff
@@ -0,0 +1,23 @@
+#' @rdname hidden_aliases
```

Comment:
This would need a corresponding `hidden.R` file (https://github.com/cran/DBI/blame/882c3921ee58d1fa97298697bf2a50161a8e401a/R/hidden.R)